code/datums/wounds/slash.dm

```diff
@@ -108,7 +121,11 @@
 	if(clot_rate < 0)
 		return BLOOD_FLOW_INCREASING
 
-/datum/wound/slash/handle_process(seconds_per_tick, times_fired)
+/datum/wound/slash/flesh/handle_process(seconds_per_tick, times_fired)
+
+	if (!victim || IS_IN_STASIS(victim))
```

Comment:
I know there's also /datum/wound/proc/on_stasis but can you double check to make sure reducing bloodflow by sutures/cauteries/whatever while someone is on stasis still demotes their wound properly? I recall that being an issue at one point

Reply:
yep it works